pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -202,6 +210,18 @@ func (o *Options) Run() error {
 		}
 		return nil
 
+	case o.ToMultiArch:
+		if cv.Spec.DesiredUpdate != nil && cv.Spec.DesiredUpdate.Architecture == configv1.ClusterVersionArchitectureMulti {
+			return fmt.Errorf("info: Update to multi cluster architecture has already been requested")
+		}
+		if err := patchDesiredUpdate(&configv1.Update{Architecture: configv1.ClusterVersionArchitectureMulti,
+			Version: cv.Status.Desired.Version}, o.Client, cv.Name); err != nil {
+
+			return err
+		}
+		fmt.Fprintln(o.Out, "Requesting update to multi cluster architecture")
```

Comment:
I realize this is an existing pattern, so feel free to stick with what you have, but I'm noticing that "Requesting ..." feels like it wants to be pre-patch logging, and I'd have expected "Requested ..." for post-patch logging.

Reply:
Agree. Changed everywhere.